src/app/public/modules/inline-delete/inline-delete.module.ts

```diff
@@ -41,9 +37,6 @@ import {
     SkyLayoutResourcesModule,
     SkyWaitModule
   ],
-  providers: [
-    SkyCoreAdapterService
```

Comment:
I moved this service to the `inline-delete` component's providers. The `ngOnDestroy` hook for the adapter service would only get called when the app is destroyed, but it looks like we want the `ngOnDestroy` lifecycle hook to execute when the `inline-delete` component is destroyed?

Reply:
Agreed. Nice catch.